essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md

```diff
@@ -1,154 +1,208 @@
-  - Start by introducing yourself in the <b>Intro's</b> section. Read carefully through the <b>welcome</b> section. You must abide by the server's rules and regulations.
-  - If you have any questions, tag the Appflowy Team or Appflowy Contributors by sending a message with a tag (<b>@AppFlowy Team </b>) for the same.
+Create a Discord account and join the [AppFlowy Community Server](https://discord.com/invite/9Q2xaN37tV). 
+
+  - Start by introducing yourself in the <b>Intro's</b> section. 
+  - Read carefully through the <b>welcome</b> section. You must abide by the server's rules and regulations.
+
+Don't be afraid to ask questions!  If you have any questions, tag the AppFlowy Team or AppFlowy Contributors by sending a message with a tag (<b>@AppFlowy Team </b>) for the same.
 
 ![image](https://user-images.githubusercontent.com/70965472/237025457-bf33e70d-8fbd-4bea-a25e-f1d0ea2147bf.png)
-*Welcome Page - Appflowy Dicord Server*
+*Welcome Page - AppFlowy Dicord Server*
+
+#### Using GitHub to Connect With the Team
 
-2. GitHub  
-  - You can get started by asking questions in the [GitHub Discussions](https://github.com/AppFlowy-IO/AppFlowy/discussions) section of Appflowy.
-  - GitHub Discussions provides a forum for Appflowy contributors to ask questions, share ideas, engage with other community members, and subsequently welcome others who join in down the road.
+You can get started by asking questions in the [GitHub Discussions](https://github.com/AppFlowy-IO/AppFlowy/discussions) section of AppFlowy.
+
+GitHub Discussions provides a forum for AppFlowy contributors to ask questions, share ideas, engage with other community members, and subsequently welcome others who join in down the road.
 
 ![image](https://user-images.githubusercontent.com/70965472/237032567-9d75b320-423e-4bc2-805d-7bcb7c96217c.png)
-*The Appflowy Discussions Page*
+*The AppFlowy Discussions Page*
 
-   - You can navigate through the various categories made available, including announcements, ideas, polls, etc.
-   - The [General Help Wanted](https://github.com/AppFlowy-IO/AppFlowy/discussions/categories/general-help-wanted) and [Technical Help Wanted](https://github.com/AppFlowy-IO/AppFlowy/discussions/categories/technical-help-wanted) sections contain open projects for first time contributors.
-   - Also, feel free to explore Appflowy's [6 Month Technical Roadmap](https://github.com/AppFlowy-IO/AppFlowy/discussions/1715), to gain a better understanding of Appflowy's current development goals over the course of the next 6 months.
+   - Navigate through the various categories, including announcements, ideas, polls, etc.
+   - Use the [General Help Wanted](https://github.com/AppFlowy-IO/AppFlowy/discussions/categories/general-help-wanted) and [Technical Help Wanted](https://github.com/AppFlowy-IO/AppFlowy/discussions/categories/technical-help-wanted) sections to find open projects for first time contributors.
```

Comment:
```suggestion
   - Use the [General Help Wanted](https://github.com/AppFlowy-IO/AppFlowy/discussions/categories/general-help-wanted) and [Technical Help Wanted](https://github.com/AppFlowy-IO/AppFlowy/discussions/categories/technical-help-wanted) section to find open projects for first time contributors.
```

Reply:
in this case the subject is plural because they are two different sections referenced